tests/common/test_runner/aws_test_runner.c

```diff
@@ -105,8 +105,8 @@ static void RunTests( void )
         IotCommon_Cleanup();
         IotMqtt_Cleanup();
 
-        RUN_TEST_GROUP( Shadow_Unit_Parser );
-        RUN_TEST_GROUP( Shadow_Unit_API );
+        //RUN_TEST_GROUP( Shadow_Unit_Parser );
```

Comment:
Is commenting out the Shadows tests intended?  Isn't it better to let them fail?